tests/smoke/test_orchestrator_smoke.py

```diff
@@ -0,0 +1,76 @@
+"""Tests the main workflow of wristpy orchestrator."""
+
+import pathlib
+
+import pytest
+
+from wristpy.core import orchestrator
+
+
+@pytest.fixture
+def test_file() -> pathlib.Path:
+    """Test data to run."""
+    return (
+        pathlib.Path(__file__).parent.parent / "sample_data" / "example_actigraph.gt3x"
+    )
+
+
+@pytest.mark.parametrize(
+    "file_name", [pathlib.Path("test_output.csv"), pathlib.Path("test_output.parquet")]
+)
+def test_happy_path(
+    file_name: pathlib.Path, tmp_path: pathlib.Path, test_file: pathlib.Path
+) -> None:
+    """Happy path for orchestrator."""
+    results = orchestrator.run(input=test_file, output=tmp_path / file_name)
+
+    assert (tmp_path / file_name).exists()
+    assert isinstance(results, orchestrator.Results)
+    assert results.enmo is not None
+    assert results.anglez is not None
+    assert results.nonwear_epoch is not None
+    assert results.sleep_windows_epoch is not None
+    assert results.physical_activity_levels is not None
+
+
+def test_different_epoch(tmp_path: pathlib.Path, test_file: pathlib.Path) -> None:
+    """Test using none default epoch."""
+    results = orchestrator.run(
+        input=test_file, output=tmp_path / "good_file.csv", epoch_length=None
+    )
+
+    assert (tmp_path / "good_file.csv").exists()
+    assert isinstance(results, orchestrator.Results)
+    assert results.enmo is not None
+    assert results.anglez is not None
+    assert results.nonwear_epoch is not None
+    assert results.sleep_windows_epoch is not None
+    assert results.physical_activity_levels is not None
+
+
+def test_bad_calibrator(test_file: pathlib.Path) -> None:
```

Comment:
Add tests for the good calibration options?

Reply:
Technically it's defaulting to using one of the good calibrators right now so this is being done in the "happy path" test. Should I add a test where this is more explicit @ReinderVosDeWael ?